src/rasberry_perception/service.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, backend=None, backend_kwargs=None, service_name=default_servi
             backend_kwargs = {}
             for arg_name in required_args:
                 p_arg = "~" + arg_name
+                rospy.logerr("Search for: {}".format( p_arg ) )
```

Comment:
No need for this rospy.logerr here this is not an error nor something to pollute a terminal with.